src/Pages/Login/Login.js

```diff
@@ -1,12 +1,51 @@
-import React, { Component } from "react";
+import React from "react";
+import "./Login.scss";
+import "../../Styles/reset.scss";
 
-class Login extends Component {
+class Login extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      checkbox: false,
+    };
   }
+  IsChecked = () => {
+    this.setState({ checkbox: !this.state.checkbox });
+    console.log(this.state.checkbox);
+  };
+
   render() {
-    return <></>;
+    return (
+      <div className="Login">
+        <div className="section-login">
+          <p className="login-title">로그인</p>
+          <form>
+            <input type="text" placeholder="아이디를 입력해주세요" className="id box" />
+            <input type="password" placeholder="비밀번호를 입력해주세요" className="password box" />
+            <div className="check-and-find">
+              <div className="checkbox-wrap">
+                <input
+                  type="checkbox"
+                  id="security"
+                  onClick={this.IsChecked}
+                  checked={this.state.checkbox}
+                />
+                <label htmlFor="security">보안접속</label>
```

Comment:
아래와 같이 label태그로 input태그를 감싸주시면 htmlFot, id 속성이 따로 필요하지 않습니다. 
```suggestion
              <label>
                <input
                  type="checkbox"
                  onClick={this.IsChecked}
                  checked={this.state.checkbox}
                />
                보안접속
              </label>
```

Reply:
오오 처음알았네요 :) 감사합니다 ㅎㅎ